src/js/components/modals/CliInstallModal.js

```diff
@@ -68,14 +68,9 @@ class CliInstallModal extends React.Component {
     }
     const clusterUrl = `${protocol}://${hostname}${port}`;
     const { selectedOS } = this.state;
-    let version = MetadataStore.parsedVersion;
-    // Prepend 'dcos-' to any version other than latest
-    if (version !== "latest") {
-      version = `dcos-${version}`;
-    }
-    const downloadUrl = `https://downloads.dcos.io/binaries/cli/${
+    const downloadUrl = `https://downloads.dcos.io/cli/testing/binaries/dcos/${
       osTypes[selectedOS]
-    }/x86-64/${version}/dcos`;
+    }/x86-64/master/dcos`;
```

Comment:
we can not merge this as it, the chance that we forget to change it after branching is too high.

Reply:
following up on slack convo and using the old code as basis, this could work 🤔 
```
if (version !== "latest" || version.includes("-dev")) {
  version = "master";
} else {
  version = "latest";
}
```

Reply:
When would `MetadataStore.parsedVersion` be `latest`? There is also another difference between the testing and stable one (testing vs. releases) : 
- `https://downloads.dcos.io/cli/testing/binaries/dcos/${osTypes[selectedOS]}/x86-64/master/dcos` (testing)
- `https://downloads.dcos.io/cli/releases/binaries/dcos/${osTypes[selectedOS]}/x86-64/latest/dcos` (stable)

Reply:
```
const downloadUrl = version.includes("-dev") ? `https://downloads.dcos.io/cli/testing/binaries/dcos/${osTypes[selectedOS]}/x86-64/master/dcos` : `https://downloads.dcos.io/cli/releases/binaries/dcos/${osTypes[selectedOS]}/x86-64/latest/dcos `;
```

Reply:
I've update it as suggested, and removed an outdated related comment.